.github/workflows/maven.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Build with Maven
       env:
         MAVEN_OPTS: -Djava.awt.headless=true -Dorg.slf4j.simpleLogger.defaultLogLevel=info -Dmaven.wagon.http.retryHandler.count=2 -Dmaven.wagon.http.pool=true
-      run: mvn test --batch-mode
+      run: mvn -PskipBrokenTestsOnUbuntu test --batch-mode
```

Comment:
doesn't really matter what this profile is called but we dont' really think it's ubuntu related..... i can replicate locally with mysql on mac ......

Reply:
its a different failing test on mac so its not the same, but as you say it doesnt matter....